tree/dataframe/src/RLoopManager.cxx

```diff
@@ -541,10 +542,25 @@ unsigned int RLoopManager::GetNextID()
    return id;
 }
 
+namespace {
+static void CheckIfPoolSizeChanged(unsigned int nSlots)
+{
+   const auto poolSize = ROOT::GetImplicitMTPoolSize();
+   if (poolSize > 0 && poolSize != nSlots) {
+      Warning("RLoopManager::Run",
+              "When the RDataFrame was constructed the size of the thread pool was %d but when starting the event loop "
```

Comment:
```suggestion
              "When the RDataFrame was constructed, the size of the thread pool was %d, but when starting the event loop, "
```

Reply:
there was one extra comma in the suggestion, added a commit with the fix

Reply:
There were three extra commas, but people will also understand without them. :-)